source/_docs/configuration/yaml.markdown

```diff
@@ -93,3 +93,13 @@ lights: !include lights.yaml
 ```
 
 More information about this feature can also be found at [splitting configuration](/docs/configuration/splitting_configuration/).
+
+### {% linkable_title Tools %}
+
+The community has found certain tools to be of use when editing YAML files.
+
+#### Notepad++
```

Comment:
Why just Notepad++? There are other more capable options, e.g., Atom, vscode.

Reply:
We shouldn't start to list editors. There are way to many to keep track and to maintain in the future.

Reply:
@arsaboo For sure, but I only have knowledge of the one I use. I figured that people could add the ones that we know work well. A starting point, if you will. :)

@fabaff Makes sense. I just see MANY people having trouble with marking up YAML. This would allow us to point newer users to a collection of solid tools, and we could update FAQ's accordingly to point to the list. That way we can point people in the right direction if they are having trouble with the correct spacing.